DOCS.txt

```diff
@@ -161,9 +151,6 @@ I really do hate writing documents.
 	and see how long you can go.
 	- Giving high priorities to kernel_team's idle thread's is always a good thing
```

Comment:
Even I was confused whether I should keep that section or not , I will now go ahead and delete . I hope that's ok ?